File: GanttView/RadGanttViewIndicatingSpecialDays/RadGanttViewIndicatingSpecialDaysVB/CustomGanttViewElement.vb
Imports Telerik.WinControls.UI

Public Class CustomGanttViewElement
    Inherits RadGanttViewElement

    Protected Overrides Function CreateGraphicalViewElement(ganttView As RadGanttViewElement) As GanttViewGraphicalViewElement
        Return New CustomGanttViewGraphicalViewElement(Me)
    End Function

    Protected Overrides ReadOnly Property ThemeEffectiveType() As Type
        Get
            Return GetType(RadGanttViewElement)
        End Get
    End Property
End Class

File: GanttView/RadGanttViewIndicatingSpecialDays/RadGanttViewIndicatingSpecialDays/CustomGanttViewElement.cs
using System;
using System.Collections.Generic;
using System.Text;
using Telerik.WinControls.UI;

namespace RadGanttViewIndicatingSpecialDays
{
    public class CustomGanttViewElement : RadGanttViewElement
    {
        protected override GanttViewGraphicalViewElement CreateGraphicalViewElement(RadGanttViewElement ganttView)
        {
            return new CustomGanttViewGraphicalViewElement(this);
        }

        protected override Type ThemeEffectiveType
        {
            get { return typeof(RadGanttViewElement); }
        }
    }
}
